src/station_cmd.cpp

```diff
@@ -518,16 +518,15 @@ CargoTypes GetEmptyMask(const Station *st)
 }
 
 /**
- * Items contains the two cargo names that are to be accepted or rejected.
- * msg is the string id of the message to display.
+ * Add news item for cargo accepted or rejected changes of a station.
+ * @param st Station of cargo change.
+ * @param cargoes Bit mask of cargo types to list.
+ * @param msg String IDk for either accepts or rejects message.
```

Comment:
```suggestion
 * @param msg String ID to show, either adding or removing cargos from the station's acceptance.
```

This might also make more sense as a `bool remove` with the String IDs specified here, instead of being passed from `UpdateStationAcceptance()`.